### PR TITLE
Nullability

### PR DESCRIPTION
**Pull request by Meeso1**

### CODE REVIEW

File: adopterAPI.json [hunk 1]
```diff
@@ -342,19 +349,18 @@
...
           "email": {
-            "type": "string",
-            "nullable": true
+            "type": "string"
           },
           "address": {
-            "$ref": "#/components/schemas/NewAddress"
+            "$ref": "#/components/schemas/NewAddress",
+            "nullable": true
           },
```
Comment (Meeso1) [inline adopterAPI.json]: Do we want `phoneNumber` and `address` to be nullable? It doesn't affect anything, so we really just need to choose what we prefer.

File: shelterAPI.json [hunk 2]
```diff
@@ -108,24 +108,19 @@
...
+            "type": "string"
           },
           "city": {
-            "type": "string",
-            "nullable": true
+            "type": "string"
           },
           "province": {
-            "type": "string",
-            "nullable": true
```
Comment (kubaurban) [inline shelterAPI.json:167]: I think it can stay nullable

File: announcementsAPI.json [hunk 3]
```diff
@@ -373,27 +374,23 @@
...
             "$ref": "#/components/schemas/Sex"
           },
           "species": {
-            "type": "string",
-            "nullable": false
+            "type": "string"
           },
           "breed": {
-            "type": "string",
-            "nullable": true
```
Comment (kubaurban) [inline announcementsAPI.json]: I think we could leave breed nullability at it is

File: announcementsAPI.json [hunk 4]
```diff
@@ -373,27 +374,23 @@
...
+            "type": "string"
           },
           "birthday": {
             "type": "string",
             "format": "date-time"
           },
           "description": {
-            "type": "string",
-            "nullable": true
```
Comment (kubaurban) [inline announcementsAPI.json]: I think we could leave description nullability at it is

File: announcementsAPI.json [hunk 5]
```diff
@@ -408,38 +405,33 @@
...
             "$ref": "#/components/schemas/Sex"
           },
           "species": {
-            "type": "string",
-            "nullable": false
+            "type": "string"
           },
           "breed": {
-            "type": "string",
-            "nullable": true
```
Comment (kubaurban) [inline announcementsAPI.json]: Same as above

File: announcementsAPI.json [hunk 6]
```diff
@@ -408,38 +405,33 @@
...
+            "type": "string"
           },
           "birthday": {
             "type": "string",
             "format": "date-time"
           },
           "description": {
-            "type": "string",
-            "nullable": true
```
Comment (kubaurban) [inline announcementsAPI.json]: Same as above

File: announcementsAPI.json
Comment (kubaurban) [inline announcementsAPI.json:1]: I would make nullable following properties:
- minAge and maxAge in GET /announcements endpoint query parameters

**Review by kubaurban — CHANGES_REQUESTED** [review]

**Review by kubaurban — APPROVED** [review]

**Pull request merged**